### ISSUE TITLE
Save last copies of gen top quark

### ISSUE THREAD

**Issue by ktht** [HEP-KBFI/tth-nanoAOD-tools#2]

**Comment by ktht** [comment]
There's nothing to be done, actually, because the gen tops are the last copies by construction: here the daughter-mother relationship is established
https://github.com/HEP-KBFI/tth-nanoAOD-tools/blob/d8fd2f54ef9bd357837ed667b47519a7864495f9/python/postprocessing/modules/genParticleProducer.py#L204-L208
And  here we select daughters that are not mothers to any tops:
https://github.com/HEP-KBFI/tth-nanoAOD-tools/blob/d8fd2f54ef9bd357837ed667b47519a7864495f9/python/postprocessing/modules/genParticleProducer.py#L212-L218

The tops that are saved must have two daughters (W + quark) which is ensured here:
https://github.com/HEP-KBFI/tth-nanoAOD-tools/blob/d8fd2f54ef9bd357837ed667b47519a7864495f9/python/postprocessing/modules/genParticleProducer.py#L236-L241

So, in conclusion, the tops that we've always extracted from the generic generator level particle collection are the last copies and therefore valid to use in top quark reweighting.
